### PR TITLE
Move the commands to a menu

### PR DESCRIPTION
**Pull request by FurkanKambay**
They take too much space...

### CODE REVIEW

File: discord-rpc-vs/DiscordRPCVSPackage.vsct [hunk 1]
```diff
@@ -1,83 +1,57 @@
...
-    called "command set" and is used to group different command inside a logically related
-    group; your package should define its own command set in order to avoid collisions
-    with command ids defined by other packages. -->
-    <!-- In this section you can define new menu groups. A menu group is a container for
-         other menus or buttons (commands); from a visual point of view you can see the
-         group as the part of a menu contained between two lines. The parent of a group
-         must be a menu. -->  
+    <Menus>
+      <Menu guid="guidDiscordRPCVSPackageCmdSet" id="SubMenu" priority="0x0100" type="Menu">
+        <Parent guid="guidSHLMainMenu" id="IDG_VS_TOOLS_CMDLINE"/>
```
Comment (FurkanKambay) [inline discord-rpc-vs/DiscordRPCVSPackage.vsct]: `IDG_VS_TOOLS_CMDLINE` is the same place with `0x0600` for me. If it's wrong, let me know. Also, how about moving it down to [`IDG_VS_TOOLS_EXT_CUST`](https://docs.microsoft.com/en-us/visualstudio/extensibility/internals/guids-and-ids-of-visual-studio-menus?view=vs-2017#tools-menu-groups) maybe?

File: discord-rpc-vs/DiscordRPCVSPackage.vsct [hunk 2]
```diff
@@ -87,19 +61,16 @@
...
-      <IDSymbol value="4132" name="cmdidToggleTimestampReset" />
-      <IDSymbol value="4133" name="cmdidToggleImagePosition" />
+      <IDSymbol value="0x0100" name="SubMenu" />
+      <IDSymbol value="0x0101" name="SubMenuGroup" />
+      <IDSymbol value="0x1021" name="cmdidToggleFileNameDisplay" />
+      <IDSymbol value="0x1022" name="cmdidToggleProjectNameDisplay" />
+      <IDSymbol value="0x1023" name="cmdidToggleTimestampDisplay" />
+      <IDSymbol value="0x1024" name="cmdidToggleTimestampReset" />
+      <IDSymbol value="0x1025" name="cmdidToggleImagePosition" />
+      <IDSymbol value="0x1026" name="cmdidTogglePresence" />
```
Comment (FurkanKambay) [inline discord-rpc-vs/DiscordRPCVSPackage.vsct]: Changed these back to hexadecimal for consistency (and I think it's convention)
If you want, I'll change the values in code, too, or change these back to decimal.

File: discord-rpc-vs/DiscordRPCVSPackage.vsct (hunk 1 above)
Comment (Swan) [reply discord-rpc-vs/DiscordRPCVSPackage.vsct]: Yeah, we can move it down there. It'd probably make more sense to do so.

File: discord-rpc-vs/DiscordRPCVSPackage.vsct (hunk 2 above)
Comment (Swan) [reply discord-rpc-vs/DiscordRPCVSPackage.vsct]: [It looks like hexadecimal is convention](https://docs.microsoft.com/en-us/visualstudio/extensibility/idsymbol-element?view=vs-2017) based on the majority of the examples provided. And yeah, preferably it should be changed in code as well for consistency's sake.

Comment (FurkanKambay) [reply discord-rpc-vs/DiscordRPCVSPackage.vsct]: @Swan I'm working on another PR, I'll do it there.

**Comment by FurkanKambay** [comment]
@Swan You're going to merge this, right? 😄

**Comment by Swan** [comment]
Yep. Thanks for your contribution. I'll update the extension accordingly.

**Pull request merged**